# Service used to set the map origin dynamically
string map_id

float32 pos_x
float32 pos_y

float32 rot_x
float32 rot_y
float32 rot_z
float32 rot_w
---